test/dotnet-new3.UnitTests/DotnetNewInstall.cs

```diff
@@ -615,7 +615,10 @@ public void CannotInstallTemplateWithoutMandatoryConfig()
         {
             var home = TestUtils.CreateTemporaryFolder("Home");
             string invalidTemplatePath = TestUtils.GetTestTemplateLocation("Invalid/MissingMandatoryConfig");
+            var assets = new string[] { Path.Combine(invalidTemplatePath, "template.json") };
+            TestUtils.EnsureTestAssetsAvailable(assets, _log);
```

Comment:
as discussed, this fix and below will probably not help. 
Let's enable debug and add more logging to try to get more info.

Reply:
ah, my fault. I will add more debug info instead.